### ISSUE TITLE
Cannot shrinkwrap because of UNMET PEER DEPENDENCY eslint@>=3.1.0

### ISSUE THREAD

**Issue by clocked0ne** [mochajs/mocha#3052]
### Prerequisites

* [x] Checked that your issue isn't already filed by cross referencing [issues with the `common mistake` label](https://github.com/mochajs/mocha/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Acommon-mistake%20)
* [x] Checked next-gen ES issues and syntax problems by using the same environment and/or transpiler configuration without Mocha to ensure it isn't just a feature that actually isn't supported in the environment in question or a bug in your code.
* [x] 'Smoke tested' the code to be tested by running it outside the real test suite to get a better sense of whether the problem is in the code under test, your usage of Mocha, or Mocha itself
* [x] Ensured that there is no discrepancy between the locally and globally installed versions of Mocha. You can find them with:
`node node_modules/.bin/mocha --version`(Local) and `mocha --version`(Global). We recommend avoiding the use of globally installed Mocha.

### Description

Cannot run  `npm shrinkwrap` because of UNMET PEER DEPENDENCY eslint@>=3.1.0 following installation.
```
npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "shrinkwrap"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0

npm ERR! Problems were encountered
npm ERR! Please correct and try again.
npm ERR! peer invalid: eslint@>=3.1.0, required by eslint-plugin-node@5.2.0
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
```

### Steps to Reproduce

Delete `node_modules` directory
Run `npm install`
Run `npm shrinkwrap`

**Expected behavior:** `npm shrinkrwap` should generate an npm-shrinkwrap.json file

**Actual behavior:** npm ERR!

**Reproduces how often:** 100%

### Versions

 If applicable, please specify:
 * The output of `mocha --version` and `node node_modules/.bin/mocha --version`: 3.5.0
 * The output of `node --version`: v7.10.0
 * The version and architecture of your operating system: Windows x64
 * Your shell (bash, zsh, PowerShell, cmd, etc.): cmd/PowerShell

**Comment by dpogue** [comment]
Will be fixed by updating `growl`, as per #3051

**Comment by boneskull** [comment]
why are you shrinkwrapping mocha's dev deps?

**Comment by boneskull** [comment]
oh... I see. bad growl.

**Comment by boneskull** [comment]
closed via d7cff37
